.github/ISSUE_TEMPLATE/new_rule.yaml

```diff
@@ -0,0 +1,88 @@
+name: New Rule
+description: Suggestions and ideas for new rules
+title: "[New Rule] Name of rule"
+labels: ["Rule: New", "Team: TRADE"]
+assignees: []
+projects: ["elastic/1268"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "Provide a detailed description of the activity to be detected."
+      placeholder: "Detailed description..."
+
+  - type: dropdown
+    id: target_ruleset
+    attributes:
+      label: Target Ruleset
+      description: "Select the target rulset."
+      options:
+        - apm
+        - cross-platform
+        - integrations
+        - linux
+        - macos
+        - ml
+        - network
+        - promotions
+        - threat_intel
+        - windows
+        - other
+
+  - type: dropdown
+    id: rule_type
+    attributes:
+      label: Target Rule Type
+      description: "Select the target type."
+      options:
+        - KQL
+        - ML_Job
+        - Threshold
+        - EQL
+        - Indicator_Match
+        - New_Terms
+        - ESQL
+
+  - type: input
+    id: ecs_version
+    attributes:
+      label: Tested ECS Version
+      description: "Specify the tested ECS version."
+      placeholder: "x.x.x"
+
+  - type: textarea
+    id: query
+    attributes:
+      label: Query
+      description: "Provide the query for the rule (optional)."
+      placeholder: "Query..."
+
+  - type: textarea
+    id: new_fields
+    attributes:
+      label: New fields required in ECS/data sources for this rule?
+      description: "List any new fields required in ECS or data sources for this rule (optional)."
+      placeholder: "New fields..."
+
+  - type: textarea
+    id: related_issues_prs
```

Comment:
Should we add a reminder that if an issue exists, link it correctly?

Reply:
This part of the form will allow them to do that. Should I add a reminder somewhere else?